hs/t/n/ui256_add_l.rsh

```diff
@@ -0,0 +1,15 @@
+'reach 0.1';
+'use strict';
+export const main = Reach.App(() => {
+  const A = Participant('A', {
+    x: UInt256,
+  });
+  init();
+  A.only(() => {
+    const x = declassify(interact.x);
+  });
+  A.publish(x);
+  const y = x + 5;
```

Comment:
Would be nice to steal Javascript's BigInt notation for specifying UInt256 literals: `5n`

Reply:
Flagged for later